scarb/Cargo.toml

```diff
@@ -76,6 +76,12 @@ xxhash-rust.workspace = true
 zip.workspace = true
 zstd.workspace = true
 
+[target.'cfg(not(unix))'.dependencies]
+reqwest = { workspace = true, default-features = true}
+
+[target.'cfg(unix)'.dependencies]
+reqwest = { workspace = true, features = ["rustls-tls-native-roots"] }
```

Comment:
Unfortunately, afaik we cannot switch based on target arch from workspace dependencies. 
Note: features are additive, so we inherit all specified in workspace manifest as well as those added here.